api/handlers/events_handler.go

```diff
@@ -0,0 +1,72 @@
+package handlers
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"github.com/pacific-theta-tau/tt-db/api/models"
+)
+
+const events_table = "events"
+
+// Helper function to scan SQL row and create new event instance
+func createEventFromRow(row *sql.Rows) (models.Event, error) {
+	var events models.Event
+	err := row.Scan(
+		&events.EventID,
+		&events.EventName,
+		&events.Category,
+		&events.EventLocation,
+		&events.EventDate,
+	)
+	if err != nil {
+		return models.Event{}, err
+	}
+
+	return events, err
+}
+
+// Get data events table
+func (h *Handler) GetAllEvents(w http.ResponseWriter, r *http.Request) {
+	fmt.Println(" - Called GetAllEvents")
+	ctx, cancel := context.WithTimeout(context.Background(), dbTimeout)
+	defer cancel()
+
+	// TODO: explicitly type columns
+	query := "SELECT * FROM " + events_table
```

Comment:
Function looking good.
Just a note: in the future, instead of querying all columns, we should explicitly write what columns to query. 
For Example, for events we don't really care about getting the `categoryID`, but rather replace that with the **name** of the category with that `categoryID` in the `eventsCategory` table. So the query would be something like:
```sql
SELECT events.eventID, events.Name, eventsCategory.categoryName, events.date, events.location
FROM events JOIN eventsCategory ON events.categoryID = eventsCategory.categoryID
```

Reply:
Yes, but it's just for the moment as we haven't finalized the category table yet. In regards to id number and such. Fully agree though.